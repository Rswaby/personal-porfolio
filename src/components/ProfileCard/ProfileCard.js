import React from 'react'
import { Card, Icon, Image } from 'semantic-ui-react'

const ProfileCard = () => (
    <Card>
        <Image src='https://res.cloudinary.com/roro-personal/image/upload/v1546480599/IMG_0258.jpg'/>
        <Card.Content>
            <Card.Header>Rohan Swaby</Card.Header>
            <Card.Description><p>Software Developer at The city College Of New York</p></Card.Description>
        </Card.Content>
        <Card.Content extra>
            <Icon name='graduation' />
            May 2019
    </Card.Content>
    </Card>
)

export default ProfileCard