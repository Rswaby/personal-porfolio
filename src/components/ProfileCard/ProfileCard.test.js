
/* eslint-disable */
import React from 'react';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { mount } from 'enzyme';
import ProfileCard from './ProfileCard'
import renderer from 'react-test-renderer';

enzyme.configure({ adapter: new Adapter() });



describe('<Navbar /> ', () => {
    it('should render correctly', () => {
        const mock = jest.fn();
        const wrapper = mount(<ProfileCard />);
    });
    test('<Navbar/> snapshot', () => {
        ///
        const component = renderer.create(<ProfileCard />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();

    })

})
