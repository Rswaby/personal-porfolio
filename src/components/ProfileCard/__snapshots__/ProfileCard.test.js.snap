// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar />  <Navbar/> snapshot 1`] = `
<div
  className="ui card"
  onClick={[Function]}
>
  <img
    className="ui image"
    src="https://res.cloudinary.com/roro-personal/image/upload/v1546480599/IMG_0258.jpg"
  />
  <div
    className="content"
  >
    <div
      className="header"
    >
      Rohan Swaby
    </div>
    <div
      className="description"
    >
      <p>
        Software Developer at The city College Of New York
      </p>
    </div>
  </div>
  <div
    className="extra content"
  >
    <i
      aria-hidden="true"
      className="graduation icon"
    />
    May 2019
  </div>
</div>
`;
